test/cmd/admin.sh

```diff
@@ -14,8 +14,8 @@ trap os::test::junit::reconcile_output EXIT
   oc delete groups/orphaned-group
   oc delete users/cascaded-user
   oc delete users/orphaned-user
-  oc delete identities/anypassword:orphaned-user
-  oc delete identities/anypassword:cascaded-user
+  oc delete identities/alwaysallow:orphaned-user
+  oc delete identities/alwaysallow:cascaded-user
```

Comment:
Why changing this?

Reply:
got the goahead from @deads2k via slack to update this to another identity. anypassword:... are apparently not available in the localup master

Reply:
Hmmm... that's weird, do you have a followup for it?